packages/xo-server-load-balancer/src/index.js

```diff
@@ -12,6 +12,8 @@ import { EXECUTION_DELAY, debug } from './utils'
 
 const PERFORMANCE_MODE = 0
 const DENSITY_MODE = 1
+const SIMPLE_MODE = 2
+const MODES = { 'Performance mode': PERFORMANCE_MODE, 'Density mode': DENSITY_MODE, 'Simple mode': SIMPLE_MODE }
```

Comment:
Use `Object.keys(MODES)` for the enum in the configuration schema to a keep a single source of truth.